week02/assignment02_test.go

```diff
@@ -0,0 +1,109 @@
+package week02
+
+import "testing"
+
+func TestArray(t *testing.T) {
+	exp := [4]string{"max", "tuto", "bombom", "tony"}
+	act := make([]string, 0)
+
+	// iterating through the array and appending
+	// to act
+	for i := range exp {
+		new := make([]string, len(act)+1)
+		copy(new, act)
+
+		act = new
+		act[len(act)-1] = exp[i]
+	}
```

Comment:
This loop is needlessly complex. We can use `range` to go through the `exp` slice which returns the current index and value of `exp`. We can then insert that data into the new slice at the correct index.

```suggestion
	for i, x := range exp {
		act[i] = x
	}
```